packages/vuetify/src/composables/icons.ts

```diff
@@ -0,0 +1,126 @@
+// Utilities
+import { computed, h, inject } from 'vue'
+
+// Components
+import { VClassIcon, VComponentIcon, VSvgIcon } from '@/components'
```

Comment:
Import from `@/components/VIcon` so we aren't pulling in every component in dev. This is recursive btw, might not be ideal.